From e7e03f129772d37137cfd46e7f309687ee462ad1 Mon Sep 17 00:00:00 2001
From: viceAdmiralGarp <vyacheslavtihonov636@gmail.com>
Date: Fri, 20 Dec 2024 19:22:54 +0200
Subject: [PATCH] modify 2 lines in firstFile.txt

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index f00c965..98d55a9 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -6,5 +6,5 @@
 6
 7
 8
-9
-10
+16
+15
-- 
2.47.0.windows.2

